client/components/thank-you/index.tsx

```diff
@@ -11,19 +11,14 @@ import React from 'react';
 import VerticalNav from 'calypso/components/vertical-nav';
 import VerticalNavItem from 'calypso/components/vertical-nav/item';
 import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
-import {
-	ThankYouNextStepProps,
-	ThankYouProps,
-	ThankYouSectionProps,
-	ThankYouThemeProps,
-} from './types';
+import { ThankYouNextStepProps, ThankYouProps, ThankYouSectionProps } from './types';
```

Comment:
Does it make sense to use `import type` here? That means the type is only imported during TS compilation and not at runtime.
```suggestion
import type { ThankYouNextStepProps, ThankYouProps, ThankYouSectionProps } from './types';
```